### ISSUE TITLE
Create links for all tabs at once

### ISSUE THREAD

**Issue by patkujawa-wf** [ku/CreateLink#19]
I would like to create links for all tabs in a window. [chitsaou/copy-as-markdown: Copying Link, Image and Tab(s) as Markdown Much Easier.](https://github.com/chitsaou/copy-as-markdown/) does this and is open source:
![image](https://cloud.githubusercontent.com/assets/6391742/14722790/a887ca78-07ca-11e6-918d-1e6ef740528d.png)

**Comment by lemartinet** [comment]
I agree, that would be great to have that option! For example when you do some research and have 10+ tabs opened and you want to save them in a document, it is kind of a pain to do it one by one.

**Comment by timotheecour** [comment]
NOTE: there is also "copy all urls" extension: https://chrome.google.com/webstore/detail/copy-all-urls/djdmadneanknadilpjiknlnanaolmbfk?hl=en
but I'd prefer if CreateLink handled it, as it has more flexible way to generate output

**Comment by dckc** [comment]
I use [TabCopy](https://chromewebstore.google.com/detail/tabcopy/micdllihgoppmejpecmkilggmaagfdmb) for this, fwiw.
